nntrainer/src/databuffer.cpp

```diff
@@ -171,6 +171,23 @@ int DataBuffer::clear(BufferType type) {
   return status;
 }
 
+int DataBuffer::clear(){
+  unsigned int i;
+
+  int status = ML_ERROR_NONE;
+  for(i = BUF_TRAIN; i <= BUF_TEST; ++i){
+    BufferType type = static_cast<BufferType>(i);
+    status = this->clear(type);
```

Comment:
Q. is it safe to do `clear()` multiple times?
For example, 
```
data_buffer->clear (BUF_TRAIN);
data_buffer->clear ();
```

Reply:
I think it is, though I will make some tests to be sure.

Reply:
Added :)

Reply:
Cool :)